### PR TITLE
Added a gradient for alarms

### LINKED ISSUE #212: Enhancement: Adding gradient to the alarm sound
### Description

I would like to add gradient to the alarm sound

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by its-me-abhishek**
### Description

Added an option in the add/update alarm view where the user can:
- Set a time limit in which the volume reaches to maximum (60s max)
- Set Minimum Volume or Starting Volume for the alarm
- Set the Maximum limit for the alarm

## Screenshots

P.S. the set volumes are visible if we tap or press their ends

## Checklist

- [ ] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [x] Code follows the established coding style guidelines
- [x] All tests are passing

### CODE REVIEW

File: lib/app/data/models/alarm_model.dart [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import 'dart:convert';
 
-import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore/cloud_firestore.dart' as firestore;
```
Comment (MarkisDev) [inline lib/app/data/models/alarm_model.dart:3]: Any reason this was done? There's no name space conflicts here right?

File: lib/app/modules/addOrUpdateAlarm/views/ascending_volume.dart [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            controller.gradient.value = gradient;
+            controller.selectedGradientDouble.value = gradient.toDouble();
+
+            return true;
+          },
+          titlePadding: const EdgeInsets.symmetric(vertical: 20, horizontal: 5),
+          backgroundColor: themeController.isLightMode.value
+              ? kLightSecondaryBackgroundColor
+              : ksecondaryBackgroundColor,
+          title: 'Volume will reach maximum in '.tr,
```
Comment (MarkisDev) [inline lib/app/modules/addOrUpdateAlarm/views/ascending_volume.dart:40]: The corresponding translation hasn't been added in utils/languages

**Review by MarkisDev — CHANGES_REQUESTED** [review]
Great work! I think you forgot to commit the translations.

File: lib/app/data/models/alarm_model.dart (hunk 1 above)
Comment (its-me-abhishek) [reply lib/app/data/models/alarm_model.dart:3]: I added this as the Query parameter that store the volumes had conflicts with the cloud_firestore and isar db, as Query is defined in both.

File: lib/app/modules/addOrUpdateAlarm/views/ascending_volume.dart (hunk 2 above)
Comment (its-me-abhishek) [reply lib/app/modules/addOrUpdateAlarm/views/ascending_volume.dart:40]: I'll add them.

**Comment by its-me-abhishek** [comment]
Added the translations, and have fixed the merge conflicts. @MarkisDev

**Review by MarkisDev — APPROVED** [review]

**Pull request merged**